pages/dev/index.tsx

```diff
@@ -109,7 +109,12 @@ function DevHandout() {
         description={
           "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
         }
-        values={["100", "200", "300", "400"]}
+        values={{
+          amount: "400",
+          area: "100",
+          car: "200",
+          people: "300",
+        }}
       />
     </>
```

Comment:
Move data to pseudodata.ts

Reply:
sa kani nash ba need pani nato? like ang dev page?

Reply:
resolved